src/functional/ffront/decorator.py

```diff
@@ -260,6 +260,36 @@ def format_itir(
             offset_provider=offset_provider,
         )
 
+    def _refactor_args_kwargs(self, *args, **kwargs) -> tuple[tuple, dict]:
+        past_params = self.past_node.params
+
+        new_args_ls = [None] * len(past_params)
+        wrong_args = [False] * len(past_params)
+        kwargs_count = 0
+        if len(past_params) < len(args) + len(kwargs):
+            raise GTTypeError(
+                "Number of parameters in function call exceeds number in function definition."
+            )
+        for param_i, param in enumerate(past_params):
+            if param.id in kwargs:
+                new_args_ls[param_i] = kwargs[param.id]
```

Comment:
```suggestion
                if param_i+1 >= len(args):
                  raise ProgramTypeError.from_past_node(
                      self.past_node, f"got multiple values for argument 'param.id''"
                  )
                new_args_ls[param_i] = kwargs[param.id]
```

For reference:
```
>>> def func(a):
...   return a
... 
>>> func(1, a=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: func() got multiple values for argument 'a'
>>>
```

Reply:
this cannot work, we can discuss on Monday

Reply:
how about this: https://github.com/GridTools/gt4py/pull/982/commits/0873010659d5d5ec3a8db3f1b76f436bd5fc8db3